src/status_im/transport/message/core.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im.transport.message.core)
+
+(def ping-topic "0x01010202")
+
+(def ttl 10000)
+
+(defprotcol StatusMessage
+  (send [this identity key topic])
+  (receive [this]))
```

Comment:
@yenda What do you thinks about having additional methods as part of the protocol, so it will look like this:
```clj
(defprotocol StatusMessage
  (send [this identity key topic])
  (receive [this])
  (ack-required? [this]
  (bulk-delivery-possible? [this])))
```

So everything related to one message type will be captured in one place and not spread in multiple namespaces. 
Messages types support bulk delivery will for example need to tread send/receive in such a way that vector of concatenated message bodies is expected and handled adequately.

Maybe it's not good to put those last two methods in the `StatusMessage` protocol, as it's a lower level concern then first two, then we can define those method in separate protocol (`TransportMessage` or `ProtocolMessage`) and create constructor functions somewhere (like `transport/create-contact-request`) which will create appropriate record implementing both protocols. 
App code then will just create particular record through those constructor functions whenever something needs to be send and call `send` protocol method on it (most probably through effects).